collab-folder/src/core/folder.rs

```diff
@@ -206,6 +206,55 @@ impl Folder {
     Some(view)
   }
 
+  pub fn move_nested_view(
```

Comment:
Add a comment to identify that it will insert the view at the front if `new_prev_id` is null.